### DISCUSSION TITLE
关于“接口响应错误 - Too Many Requests”

### DISCUSSION THREAD

**Discussion by weilunn** [yetone/bob-plugin-openai-translator#84] [Q&A] [answered]
您好！

我最近使用这个插件时，总是提示”接口响应错误 - Too Many Requests“。请问该怎么解决呢？

感谢您的开发！

**Comment by liby** [comment, marked as answer]
由于频繁收到相同的 issue（实则并不是由 [bob-plugin-openai-translator](https://github.com/openai-translator/bob-plugin-openai-translator) 导致的）：

- https://github.com/openai-translator/bob-plugin-openai-translator/issues/47#issuecomment-1573030116

- https://github.com/openai-translator/bob-plugin-openai-translator/issues/85

- https://github.com/openai-translator/bob-plugin-openai-translator/issues/87

- https://github.com/openai-translator/bob-plugin-openai-translator/issues/91

- https://github.com/openai-translator/bob-plugin-openai-translator/issues/93

目前在[最新版本](https://github.com/openai-translator/bob-plugin-openai-translator/releases/tag/v1.1.2)中已经将 429 Too Many Requests 的提示信息进行优化，欢迎大家更新体验～

<img width="399" alt="image" src="https://github.com/openai-translator/bob-plugin-openai-translator/assets/38807139/77a4e872-67e9-4ef1-b60a-16564177bc1c">
